package.json

```diff
@@ -113,5 +113,11 @@
     "webpack": "^5.88.1",
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2"
+  },
+  "resolutions": {
+    "nan": "2.17.0"
```

Comment:
It was necessary to pin the "nan" package to this newer version because otherwise it broke the `npm install` of binaries on the newer node version.